03-intro-telemetry-data.qmd

```diff
@@ -246,18 +244,20 @@ Now we have clean data to work with we can get to the fun data exploration part!
 ```{r read in geopackage}
 
 #bou = read.csv("clean_data/caribou.csv")
+
+# or 
+
 bou_sf = st_read("clean_data/caribou.gpkg")
```

Comment:
Ditto above for `read_sf()` instead of `st_read()`?

Reply:
I always use st_read... but not sure I know the difference? Im happy to adjust to read_sf if it makes sense.

Reply:
I think the main difference is that it returns a tibble rather than regular `sf` `data.frame` (again, mostly giving the nicer print method)